mod/metrics/set/all/colours.rb

```diff
@@ -35,12 +35,24 @@ def mono_colour
     ]
   end
 
+  # Colours for non-numbers
+  def light_colour
+    [
+      "#cccccc",
+      "#e4e9f1"
+    ]
+  end
+
   def colorify value, mono=false
     return "" if value == ""
-    colour = pick_colour(value, mono)
+    colour = numeric?(value) ? pick_colour(value, mono) : light_colour[0]
     wrap_color_div(value, colour)
   end
 
+  def numeric? value
```

Comment:
There is already a `number?` methods for strings

Reply:
Ah, I wasn't aware